Documentation/getting-started.md

```diff
@@ -1,22 +1,56 @@
-# Getting Started with Media SDK
+## Getting started
 
-## Before starting
+The Adobe Experience Platform Media for Edge Network mobile extension has the following dependencies, which must be installed prior to installing the extension:
+- [MobileCore](https://github.com/adobe/aepsdk-core-android)
+- [Edge](https://github.com/adobe/aepsdk-edge-android)
+- [EdgeIdentity](https://github.com/adobe/aepsdk-edgeidentity-android)
 
-Media extension depends on the following extensions:
-* [Mobile Core and Identity](https://github.com/adobe/aepsdk-core-android)
-* [Analytics](https://github.com/adobe/aepsdk-analytics-android) (peer dependency)
+## Configuration
+
+### Configure Dependencies
+Configure the Edge, EdgeIdentity extensions in the mobile property using the Data Collection UI.
+
+> **Note** 
+> If this is your first time setting up Edge extensions and using Data Collection UI, please follow this [tutorial](https://github.com/adobe/aepsdk-edge-android/tree/main/Documentation/Tutorials) to learn about Adobe Experience Platform and how to setup required schemas, datasets, datastreams and creating mobile property etc.
+
+### Configure EdgeMedia extension
+Currently EdgeMedia doesn't have a Data Collection extension and needs to be configured programmatically.
+
+#### Configuration Keys
+| Name | Key | Value | Required |
+| --- | --- | --- | --- |
+| Channel | "edgemedia.channel" | String | **Yes** |
+| Player Name | "edgemedia.playerName" | String | **Yes** |
+| Application Version | "edgemedia.appVersion" | String | **No** |
+
+##### Java 
+    ```java
+    Map<String, Object> mediaConfiguration = new HashMap<>();
+    mediaConfiguration.put("edgemedia.channel", "<YOUR_CHANNEL_NAME>");
+    mediaConfiguration.put("edgemedia.playerName", "<YOUR_PLAYER_NAME>");
+    mediaConfiguration.put("edgemedia.appVersion", "<YOUR_APP_VERSION>");
+
+    MobileCore.updateConfiguration(mediaConfiguration);
+    ```
+
+##### Kotlin
+    ```koltin
+    val mediaConfiguration = mapOf("edgemedia.channel" to "<YOUR_CHANNEL_NAME>", "edgemedia.playerName" to "<YOUR_PLAYER_NAME>", "edgemedia.appVersion" to "<YOUR_APP_VERSION>") as Map<String, Any>
```

Comment:
nit: can you place each map property on a new line for readability, such as 
```
val mediaConfiguration = mapOf<String, Any>(
    "edgemedia.channel" to "<YOUR_CHANNEL_NAME>", 
    "edgemedia.playerName" to "<YOUR_PLAYER_NAME>", 
    "edgemedia.appVersion" to "<YOUR_APP_VERSION>"
)
```
Also, remove the  `as Map<String, Any>` and add the types when calling `mapOf<String, Any>(...)`